easybuild/tools/options.py

```diff
@@ -215,12 +215,16 @@ def __init__(self, *args, **kwargs):
         # set up constants to seed into config files parser, by section
         self.go_cfg_constants = {
             self.DEFAULTSECT: {
-                'DEFAULT_REPOSITORYPATH': (self.default_repositorypath[0], "Default easyconfigs repository path"),
+                'DEFAULT_REPOSITORYPATH': (self.default_repositorypath[0], 
+                                           "Default easyconfigs repository path"),
                 'DEFAULT_ROBOT_PATHS': (os.pathsep.join(self.default_robot_paths),
                                         "List of default robot paths ('%s'-separated)" % os.pathsep),
+                'USER': (os.environ['USER'], "Current username, derived from the os environment"),
+                'HOME': (os.environ['HOME'], 
```

Comment:
@brevans please use `os.path.expanduser('~')` here rather than `os.environ['HOME']`

There's a subtle difference, and we use `expanduser` in other places already, so let's be consistent; this is still based on `$HOME` on Linux though, see:

```shell
$ HOME=foo python -c "import os; print os.path.expanduser('~')"
foo
```